fundamentals/fundamentals-collections/src/main/scala/izumi/functional/IzEitherAggregations.scala

```diff
@@ -0,0 +1,196 @@
+package izumi.functional
+
+import izumi.functional.IzEitherAggregations.*
+import izumi.fundamentals.collections.nonempty.NonEmptyList
+
+import scala.collection.compat.*
+import scala.language.implicitConversions
+
+trait IzEitherAggregations {
+  @inline implicit final def EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], R]]
+  ): EitherBiAggregate[L, R, Col] = new EitherBiAggregate(col)
+  @inline implicit final def EitherBiFlatAggregate[L, R, Col[x] <: IterableOnce[x], Col2[x] <: IterableOnce[x]](
+    col: Col[Either[NonEmptyList[L], Col2[R]]]
+  ): EitherBiFlatAggregate[L, R, Col, Col2] = new EitherBiFlatAggregate(col)
+  @inline implicit final def EitherBiFind[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFind[Col, T] = new EitherBiFind(col)
+  @inline implicit final def EitherBiFoldLeft[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFoldLeft[Col, T] = new EitherBiFoldLeft(col)
+  @inline implicit final def EitherBiMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiMapAggregate[Col, T] = new EitherBiMapAggregate(col)
+  @inline implicit final def EitherBiFlatMapAggregate[Col[x] <: IterableOnce[x], T](col: Col[T]): EitherBiFlatMapAggregate[Col, T] = new EitherBiFlatMapAggregate(col)
+}
+
+object IzEitherAggregations extends IzEitherAggregations {
+
+  final class EitherBiAggregate[L, R, Col[x] <: IterableOnce[x]](private val col: Col[Either[NonEmptyList[L], R]]) extends AnyVal {
+    /** `sequence` with error accumulation */
+    def biAggregate(implicit b: Factory[R, Col[R]]): Either[NonEmptyList[L], Col[R]] = {
+      val bad = List.newBuilder[L]
```

Comment:
@YaroslavSahach 
These can all reuse existing List-returning functions in IzEither, no need to copy full impls.
```suggestion
      IzEither.EitherBiAggregate(col)
        .biAggregate
        .left.map(NonEmptyList.unsafeFrom)
```